packages/airnode-node/src/api/index.ts

```diff
@@ -184,6 +185,16 @@ export function isPerformApiCallFailure(
   return !!(value as ApiCallErrorResponse).errorMessage;
 }
 
+export function errorMsgFromAxiosError(e: AxiosError): string {
+  if (e.response) {
+    return `with status code ${e.response.status}`;
+  } else if (e.request) {
+    return 'with no response';
+  } else {
+    return 'in building the request';
```

Comment:
Hmm. At first glance at the error message, I thought that the airnode-adapter was not able to build a request based on the OIS. But that's not what the message is trying to convey. But I can't come up with a better wording tbh :sweat_smile: Maybe I'm just reading too much into it. WDYT?

Reply:
The [axios error handling docs](https://axios-http.com/docs/handling_errors) don't help us out much either 😅 
```ts
  } else {
      // Something happened in setting up the request that triggered an Error
      console.log('Error', error.message);
  }
```

How about: 'in building the *http* request`?

Reply:
Maybe `while establishing the connection`? But `in building the HTTP request` is fine as well. I wouldn't mention the Axios specifically though.